tests/sceneManager_test.cpp

```diff
@@ -11,6 +11,68 @@ using objects::Object;
 using objects::TriangleObj;
 
 const double kSkipFreq = 1000;
+const core::Vec3 kSkipVec3(0, 0, 0);
 generators::FakeRandomGen fakeRand;
 
-// TODO: Change all doubles to float
+class SceneManagerTest : public ::testing::Test
+{
+protected:
+    SceneManager manager;
+    std::vector<objects::EnergyCollector *> energyCollectors;
+    double energyCollectorRadius = objects::EnergyCollector(kSkipVec3).getRadius();
```

Comment:
This is a sign of bad design, and hardcoded dependencies and constant numbers. Please fix the other comment about passing the radius (or something) via constructor.

Code you write needs to be generic and work on many different parameters. This way also you will be able to pass easier cases, like say 3 collectors.

Reply:
noted